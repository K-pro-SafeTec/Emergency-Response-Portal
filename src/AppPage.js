import React from 'react';
import AppBar from '@material-ui/core/AppBar';
import Toolbar from '@material-ui/core/Toolbar';
import Typography from '@material-ui/core/Typography';

export default ({ title, Component }) => (
  <React.Fragment>
    <AppBar position="static" color="default">
      <Toolbar>
        <Typography variant="title" color="inherit">{title}</Typography>
      </Toolbar>
    </AppBar>
    <Component />
  </React.Fragment>
);
