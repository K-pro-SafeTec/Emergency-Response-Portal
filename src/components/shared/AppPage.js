import React from 'react';
import AppBar from '@material-ui/core/AppBar';
import Toolbar from '@material-ui/core/Toolbar';
import Typography from '@material-ui/core/Typography';
import IconButton from '@material-ui/core/IconButton';
import ArrowBack from '@material-ui/icons/ArrowBack';
import Home from '@material-ui/icons/Home';
import SafetecLogo from '../../icons/safetek_for_now.svg';
import { withStyles } from '@material-ui/core';
import { Link } from 'react-router-dom';

const styles = {
  grow: {
    flexGrow: '1',
  },
  homeButton: {
    marginLeft: '-12px',
  },
  backButton: {
    marginRight: '16px',
  },
};

const AppPage = ({ classes, title, back, children }) => (
  <React.Fragment>
    <AppBar position="static" color="default">
      <Toolbar>
        {back && (
          <div>
            <IconButton className={classes.homeButton} component={Link} to="/">
              <Home />
            </IconButton>
            <IconButton className={classes.backButton} component={Link} to={back}>
              <ArrowBack />
            </IconButton>
          </div>
        )}
        <Typography variant="title" color="inherit" className={classes.grow}>
          {title}
        </Typography>
        <a href="https://www.safetec.no/">
          <img
            src={SafetecLogo}
            alt="Levert av Safetec"
            height="40px"
          />
        </a>
      </Toolbar>
    </AppBar>
    {children}
  </React.Fragment>
);

export default withStyles(styles)(AppPage);
