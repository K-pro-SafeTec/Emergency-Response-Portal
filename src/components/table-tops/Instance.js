import React from 'react';
import Typography from '@material-ui/core/Typography';
import List from '@material-ui/core/List';
import ListItem from '@material-ui/core/ListItem';
import ListItemIcon from '@material-ui/core/ListItemIcon';
import ListItemText from '@material-ui/core/ListItemText';
import Paper from '@material-ui/core/Paper';
import People from '@material-ui/icons/People';
import { Link } from 'react-router-dom';
import AppPage from '../shared/AppPage';
import EntityInfo from '../shared/EntityInfo';
import IconTableTop from '../../icons/main-menu/tableTops.svg';
import { tableTopScenarioById } from '../../dummy-data/tableTopScenario';
import { getTableTopInstanceByKey } from '../../dummy-data/tableTopInstance';
import { teamById } from '../../dummy-data/team';


const Instance = ({ match }) => {
  const instance = getTableTopInstanceByKey(match.params.scenarioId, match.params.date);
  if (instance) {
    const scenario = tableTopScenarioById[match.params.scenarioId];
    return (
      <AppPage title="Table top" back="../../../teams/">
        <Paper style={{margin: '1em', padding: '1em'}}>
          <EntityInfo Icon={(props) => <img src={IconTableTop} alt="Table Top" {...props} />}>
            <Typography variant="headline">DFU {scenario.dfuNr}</Typography>
            <Typography variant="subheading">{scenario.dfuTitle}</Typography>
          </EntityInfo>
          <Typography variant="subheading" gutterBottom>Scenario:</Typography>
          <Typography variant="body1" paragraph>{scenario.scenario}</Typography>
          <Typography variant="subheading" gutterBottom>Scenariobeskrivelse:</Typography>
          <Typography variant="body1" paragraph>{scenario.description}</Typography>
          <Typography variant="subheading" gutterBottom>Utvikling:</Typography>
          <Typography variant="body1" paragraph>{instance.development}</Typography>
        </Paper>
        <List>
          <ListItem>
            <Typography variant="title">Deltakerlag</Typography>
          </ListItem>
          {scenario.teams.map(teamId => {
            const team = teamById[teamId];
            return (
              <ListItem
                key={team.id}
                button
                component={Link}
                to={`../../../teams/${team.id}/`}
              >
                <ListItemIcon>
                  <People />
                </ListItemIcon>
                <ListItemText>
                  {team.name}
                </ListItemText>
              </ListItem>
            );
          })}
        </List>
      </AppPage>
    );
  }
  return (
    <AppPage title="Table top" back="..">
      <Typography>Fant ikke table top</Typography>
    </AppPage>
  )
};

export default Instance;
