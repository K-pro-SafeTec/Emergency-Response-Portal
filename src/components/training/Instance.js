import React from 'react';
import Typography from '@material-ui/core/Typography';
import List from '@material-ui/core/List';
import ListItem from '@material-ui/core/ListItem';
import ListItemIcon from '@material-ui/core/ListItemIcon';
import ListItemText from '@material-ui/core/ListItemText';
import Paper from '@material-ui/core/Paper';
import People from '@material-ui/icons/People';
import { Link } from 'react-router-dom';
import AppPage from '../shared/AppPage';
import EntityInfo from '../shared/EntityInfo';
import IconTraining from '../../icons/main-menu/training.svg';
import { trainingScenarioById } from '../../dummy-data/trainingScenario';
import { getTrainingInstanceByKey } from '../../dummy-data/trainingInstance';
import { teamById } from '../../dummy-data/team';


const Instance = ({ match }) => {
  const instance = getTrainingInstanceByKey(match.params.scenarioId, match.params.date);
  if (instance) {
    const scenario = trainingScenarioById[match.params.scenarioId];
    return (
      <AppPage title="Trening" back="../../../teams/">
        <Paper style={{margin: '1em', padding: '1em'}}>
          <EntityInfo Icon={(props) => <img src={IconTraining} alt="Trening" {...props} />}>
            <Typography variant="headline">{scenario.theme}</Typography>
            <Typography variant="subheading">{instance.date}</Typography>
          </EntityInfo>
          <Typography variant="subheading" gutterBottom>MÃ¥l:</Typography>
          <Typography variant="body1" paragraph>{scenario.goals}</Typography>
          <Typography variant="subheading" gutterBottom>Oppsett:</Typography>
          <Typography variant="body1" paragraph>{scenario.setup}</Typography>
        </Paper>
        <List>
          <ListItem>
            <Typography variant="title">Deltakerlag</Typography>
          </ListItem>
          {scenario.participatingTeams.map(teamId => {
            const team = teamById[teamId];
            return (
              <ListItem
                key={team.id}
                button
                component={Link}
                to={`../../../teams/${team.id}/`}
              >
                <ListItemIcon>
                  <People />
                </ListItemIcon>
                <ListItemText>
                  {team.name}
                </ListItemText>
              </ListItem>
            );
          })}
        </List>
      </AppPage>
    );
  }
  return (
    <AppPage title="Trening" back="..">
      <Typography>Fant ikke trening</Typography>
    </AppPage>
  )
};

export default Instance;
