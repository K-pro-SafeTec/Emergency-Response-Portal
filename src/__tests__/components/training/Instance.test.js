import React from 'react';
import TestRenderer from 'react-test-renderer';
import { MemoryRouter as Router } from 'react-router-dom'
import AppPage from "../../../components/shared/AppPage";
import Instance from "../../../components/training/Instance";

let testRenderer;
let testInstance;

beforeAll(() => {
  testRenderer = TestRenderer.create(
    <Router>
      <Instance
        match={
          {params:
              {
                scenarioId: 0,
                date: '2018-09-17'
              }
          }
        }
      />
    </Router>);
  testInstance = testRenderer.root;
});

test('renders correctly', () => {
  const tree = testRenderer.toJSON();
  expect(tree).toMatchSnapshot();
});

test('title is correct', () => {
  expect(testInstance.findByType(AppPage).props.title).toBe("Trening");
});
