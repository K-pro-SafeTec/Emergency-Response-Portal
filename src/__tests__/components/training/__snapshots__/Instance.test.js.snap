// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
Array [
  <header
    className="MuiPaper-root-14 MuiPaper-elevation4-20 MuiAppBar-root-5 MuiAppBar-positionFixed-6 MuiAppBar-colorDefault-11 mui-fixed"
  >
    <div
      className="MuiToolbar-root-41 MuiToolbar-regular-43 MuiToolbar-gutters-42"
    >
      <a
        className="MuiButtonBase-root-51 MuiIconButton-root-45 AppPage-homeButton-2"
        href="/"
        onBlur={[Function]}
        onClick={[Function]}
        onFocus={[Function]}
        onKeyDown={[Function]}
        onKeyUp={[Function]}
        onMouseDown={[Function]}
        onMouseLeave={[Function]}
        onMouseUp={[Function]}
        onTouchEnd={[Function]}
        onTouchMove={[Function]}
        onTouchStart={[Function]}
        role="button"
        tabIndex="0"
      >
        <span
          className="MuiIconButton-label-50"
        >
          <svg
            aria-hidden="true"
            className="MuiSvgIcon-root-54"
            focusable="false"
            role="presentation"
            viewBox="0 0 24 24"
          >
            <path
              d="M10 20v-6h4v6h5v-8h3L12 3 2 12h3v8z"
            />
            <path
              d="M0 0h24v24H0z"
              fill="none"
            />
          </svg>
        </span>
        <span
          className="MuiTouchRipple-root-134"
        />
      </a>
      <button
        className="MuiButtonBase-root-51 MuiButtonBase-disabled-52 MuiIconButton-root-45 MuiIconButton-disabled-49"
        disabled={true}
        onBlur={[Function]}
        onClick={[Function]}
        onFocus={[Function]}
        onKeyDown={[Function]}
        onKeyUp={[Function]}
        onMouseDown={[Function]}
        onMouseLeave={[Function]}
        onMouseUp={[Function]}
        onTouchEnd={[Function]}
        onTouchMove={[Function]}
        onTouchStart={[Function]}
        tabIndex="-1"
        type="button"
      >
        <span
          className="MuiIconButton-label-50"
        >
          <svg
            aria-hidden="true"
            className="MuiSvgIcon-root-54"
            focusable="false"
            role="presentation"
            viewBox="0 0 24 24"
          >
            <path
              d="M0 0h24v24H0z"
              fill="none"
            />
            <path
              d="M20 11H7.83l5.59-5.59L12 4l-8 8 8 8 1.41-1.41L7.83 13H20v-2z"
            />
          </svg>
        </span>
      </button>
      <button
        className="MuiButtonBase-root-51 MuiButtonBase-disabled-52 MuiIconButton-root-45 MuiIconButton-disabled-49"
        disabled={true}
        onBlur={[Function]}
        onClick={[Function]}
        onFocus={[Function]}
        onKeyDown={[Function]}
        onKeyUp={[Function]}
        onMouseDown={[Function]}
        onMouseLeave={[Function]}
        onMouseUp={[Function]}
        onTouchEnd={[Function]}
        onTouchMove={[Function]}
        onTouchStart={[Function]}
        tabIndex="-1"
        type="button"
      >
        <span
          className="MuiIconButton-label-50"
        >
          <svg
            aria-hidden="true"
            className="MuiSvgIcon-root-54"
            focusable="false"
            role="presentation"
            viewBox="0 0 24 24"
          >
            <path
              d="M0 0h24v24H0z"
              fill="none"
            />
            <path
              d="M12 4l-1.41 1.41L16.17 11H4v2h12.17l-5.58 5.59L12 20l8-8z"
            />
          </svg>
        </span>
      </button>
      <h2
        className="MuiTypography-root-63 MuiTypography-title-69 MuiTypography-colorInherit-92 AppPage-grow-1"
      >
        Trening
      </h2>
      <a
        className="AppPage-safetecLink-3"
        href="https://www.safetec.no/"
      >
        <img
          alt="Levert av Safetec"
          height="44px"
          src="safetec.svg"
          width="153px"
        />
      </a>
    </div>
  </header>,
  <div
    className="AppPage-content-4"
  >
    <div
      className="MuiPaper-root-14 MuiPaper-elevation2-18 MuiPaper-rounded-15"
      style={
        Object {
          "margin": "1em",
          "padding": "1em",
        }
      }
    >
      <div
        className="EntityInfo-container-98"
      >
        <img
          alt="Trening"
          className="EntityInfo-avatar-99"
          src="training.svg"
        />
        <div
          className="EntityInfo-info-100"
        >
          <h1
            className="MuiTypography-root-63 MuiTypography-headline-68"
          >
            Utstyrstrening
          </h1>
          <h3
            className="MuiTypography-root-63 MuiTypography-subheading-70"
          />
        </div>
      </div>
      <h3
        className="MuiTypography-root-63 MuiTypography-subheading-70 MuiTypography-gutterBottom-90"
      >
        Mål:
      </h3>
      <p
        className="MuiTypography-root-63 MuiTypography-body1-72 MuiTypography-paragraph-91"
      >
        Være kjent med eget utstyr  og bruken av dette
      </p>
      <h3
        className="MuiTypography-root-63 MuiTypography-subheading-70 MuiTypography-gutterBottom-90"
      >
        Oppsett:
      </h3>
      <p
        className="MuiTypography-root-63 MuiTypography-body1-72"
      >
        Påkledning av innsatsbekledning
      </p>
      <p
        className="MuiTypography-root-63 MuiTypography-body1-72"
      >
        Gjennomgang av bruk av røykdykkerapparat
      </p>
      <p
        className="MuiTypography-root-63 MuiTypography-body1-72"
      >
        Bytte av luftflaske - røykdykkerapparat
      </p>
      <p
        className="MuiTypography-root-63 MuiTypography-body1-72"
      >
        Gjennomgang og test av kommunikasjon
      </p>
      <p
        className="MuiTypography-root-63 MuiTypography-body1-72"
      >
        Bli kjent runde ved behov
      </p>
      <div
        className="PersonList-root-101"
      >
        <h3
          className="MuiTypography-root-63 MuiTypography-subheading-70 MuiTypography-gutterBottom-90"
          style={
            Object {
              "marginTop": "24px",
            }
          }
        >
          Deltakere:
        </h3>
        <ul
          className="MuiList-root-103 MuiList-padding-104"
        >
          <div
            className="MuiButtonBase-root-51 MuiListItem-root-107 MuiListItem-default-110 MuiListItem-gutters-114 MuiListItem-button-115 PersonList-listItem-102"
            onBlur={[Function]}
            onClick={[Function]}
            onFocus={[Function]}
            onKeyDown={[Function]}
            onKeyUp={[Function]}
            onMouseDown={[Function]}
            onMouseLeave={[Function]}
            onMouseUp={[Function]}
            onTouchEnd={[Function]}
            onTouchMove={[Function]}
            onTouchStart={[Function]}
            role="button"
            tabIndex="0"
          >
            <span
              className="MuiButtonBase-root-51 MuiIconButton-root-45 MuiPrivateSwitchBase-root-124 MuiCheckbox-root-118 MuiCheckbox-colorPrimary-122"
              onBlur={[Function]}
              onFocus={[Function]}
              onKeyDown={[Function]}
              onKeyUp={[Function]}
              onMouseDown={[Function]}
              onMouseLeave={[Function]}
              onMouseUp={[Function]}
              onTouchEnd={[Function]}
              onTouchMove={[Function]}
              onTouchStart={[Function]}
              tabIndex={null}
            >
              <span
                className="MuiIconButton-label-50"
              >
                <svg
                  aria-hidden="true"
                  className="MuiSvgIcon-root-54"
                  focusable="false"
                  role="presentation"
                  viewBox="0 0 24 24"
                >
                  <path
                    d="M19 5v14H5V5h14m0-2H5c-1.1 0-2 .9-2 2v14c0 1.1.9 2 2 2h14c1.1 0 2-.9 2-2V5c0-1.1-.9-2-2-2z"
                  />
                </svg>
                <input
                  checked={false}
                  className="MuiPrivateSwitchBase-input-127"
                  data-indeterminate={false}
                  onChange={[Function]}
                  tabIndex={-1}
                  type="checkbox"
                />
              </span>
            </span>
            <div
              className="MuiListItemText-root-128"
            >
              <span
                className="MuiTypography-root-63 MuiTypography-subheading-70 MuiListItemText-primary-131"
              >
                Marie Danielsen
              </span>
            </div>
            <span
              className="MuiTouchRipple-root-134"
            />
          </div>
          <div
            className="MuiButtonBase-root-51 MuiListItem-root-107 MuiListItem-default-110 MuiListItem-gutters-114 MuiListItem-button-115 PersonList-listItem-102"
            onBlur={[Function]}
            onClick={[Function]}
            onFocus={[Function]}
            onKeyDown={[Function]}
            onKeyUp={[Function]}
            onMouseDown={[Function]}
            onMouseLeave={[Function]}
            onMouseUp={[Function]}
            onTouchEnd={[Function]}
            onTouchMove={[Function]}
            onTouchStart={[Function]}
            role="button"
            tabIndex="0"
          >
            <span
              className="MuiButtonBase-root-51 MuiIconButton-root-45 MuiPrivateSwitchBase-root-124 MuiCheckbox-root-118 MuiCheckbox-colorPrimary-122"
              onBlur={[Function]}
              onFocus={[Function]}
              onKeyDown={[Function]}
              onKeyUp={[Function]}
              onMouseDown={[Function]}
              onMouseLeave={[Function]}
              onMouseUp={[Function]}
              onTouchEnd={[Function]}
              onTouchMove={[Function]}
              onTouchStart={[Function]}
              tabIndex={null}
            >
              <span
                className="MuiIconButton-label-50"
              >
                <svg
                  aria-hidden="true"
                  className="MuiSvgIcon-root-54"
                  focusable="false"
                  role="presentation"
                  viewBox="0 0 24 24"
                >
                  <path
                    d="M19 5v14H5V5h14m0-2H5c-1.1 0-2 .9-2 2v14c0 1.1.9 2 2 2h14c1.1 0 2-.9 2-2V5c0-1.1-.9-2-2-2z"
                  />
                </svg>
                <input
                  checked={false}
                  className="MuiPrivateSwitchBase-input-127"
                  data-indeterminate={false}
                  onChange={[Function]}
                  tabIndex={-1}
                  type="checkbox"
                />
              </span>
            </span>
            <div
              className="MuiListItemText-root-128"
            >
              <span
                className="MuiTypography-root-63 MuiTypography-subheading-70 MuiListItemText-primary-131"
              >
                Lars Arnesen
              </span>
            </div>
            <span
              className="MuiTouchRipple-root-134"
            />
          </div>
          <div
            className="MuiButtonBase-root-51 MuiListItem-root-107 MuiListItem-default-110 MuiListItem-gutters-114 MuiListItem-button-115 PersonList-listItem-102"
            onBlur={[Function]}
            onClick={[Function]}
            onFocus={[Function]}
            onKeyDown={[Function]}
            onKeyUp={[Function]}
            onMouseDown={[Function]}
            onMouseLeave={[Function]}
            onMouseUp={[Function]}
            onTouchEnd={[Function]}
            onTouchMove={[Function]}
            onTouchStart={[Function]}
            role="button"
            tabIndex="0"
          >
            <span
              className="MuiButtonBase-root-51 MuiIconButton-root-45 MuiPrivateSwitchBase-root-124 MuiCheckbox-root-118 MuiCheckbox-colorPrimary-122"
              onBlur={[Function]}
              onFocus={[Function]}
              onKeyDown={[Function]}
              onKeyUp={[Function]}
              onMouseDown={[Function]}
              onMouseLeave={[Function]}
              onMouseUp={[Function]}
              onTouchEnd={[Function]}
              onTouchMove={[Function]}
              onTouchStart={[Function]}
              tabIndex={null}
            >
              <span
                className="MuiIconButton-label-50"
              >
                <svg
                  aria-hidden="true"
                  className="MuiSvgIcon-root-54"
                  focusable="false"
                  role="presentation"
                  viewBox="0 0 24 24"
                >
                  <path
                    d="M19 5v14H5V5h14m0-2H5c-1.1 0-2 .9-2 2v14c0 1.1.9 2 2 2h14c1.1 0 2-.9 2-2V5c0-1.1-.9-2-2-2z"
                  />
                </svg>
                <input
                  checked={false}
                  className="MuiPrivateSwitchBase-input-127"
                  data-indeterminate={false}
                  onChange={[Function]}
                  tabIndex={-1}
                  type="checkbox"
                />
              </span>
            </span>
            <div
              className="MuiListItemText-root-128"
            >
              <span
                className="MuiTypography-root-63 MuiTypography-subheading-70 MuiListItemText-primary-131"
              >
                Tor Antonsen
              </span>
            </div>
            <span
              className="MuiTouchRipple-root-134"
            />
          </div>
          <div
            className="MuiButtonBase-root-51 MuiListItem-root-107 MuiListItem-default-110 MuiListItem-gutters-114 MuiListItem-button-115 PersonList-listItem-102"
            onBlur={[Function]}
            onClick={[Function]}
            onFocus={[Function]}
            onKeyDown={[Function]}
            onKeyUp={[Function]}
            onMouseDown={[Function]}
            onMouseLeave={[Function]}
            onMouseUp={[Function]}
            onTouchEnd={[Function]}
            onTouchMove={[Function]}
            onTouchStart={[Function]}
            role="button"
            tabIndex="0"
          >
            <span
              className="MuiButtonBase-root-51 MuiIconButton-root-45 MuiPrivateSwitchBase-root-124 MuiCheckbox-root-118 MuiCheckbox-colorPrimary-122"
              onBlur={[Function]}
              onFocus={[Function]}
              onKeyDown={[Function]}
              onKeyUp={[Function]}
              onMouseDown={[Function]}
              onMouseLeave={[Function]}
              onMouseUp={[Function]}
              onTouchEnd={[Function]}
              onTouchMove={[Function]}
              onTouchStart={[Function]}
              tabIndex={null}
            >
              <span
                className="MuiIconButton-label-50"
              >
                <svg
                  aria-hidden="true"
                  className="MuiSvgIcon-root-54"
                  focusable="false"
                  role="presentation"
                  viewBox="0 0 24 24"
                >
                  <path
                    d="M19 5v14H5V5h14m0-2H5c-1.1 0-2 .9-2 2v14c0 1.1.9 2 2 2h14c1.1 0 2-.9 2-2V5c0-1.1-.9-2-2-2z"
                  />
                </svg>
                <input
                  checked={false}
                  className="MuiPrivateSwitchBase-input-127"
                  data-indeterminate={false}
                  onChange={[Function]}
                  tabIndex={-1}
                  type="checkbox"
                />
              </span>
            </span>
            <div
              className="MuiListItemText-root-128"
            >
              <span
                className="MuiTypography-root-63 MuiTypography-subheading-70 MuiListItemText-primary-131"
              >
                Fredrik Birkeland
              </span>
            </div>
            <span
              className="MuiTouchRipple-root-134"
            />
          </div>
          <div
            className="MuiButtonBase-root-51 MuiListItem-root-107 MuiListItem-default-110 MuiListItem-gutters-114 MuiListItem-button-115 PersonList-listItem-102"
            onBlur={[Function]}
            onClick={[Function]}
            onFocus={[Function]}
            onKeyDown={[Function]}
            onKeyUp={[Function]}
            onMouseDown={[Function]}
            onMouseLeave={[Function]}
            onMouseUp={[Function]}
            onTouchEnd={[Function]}
            onTouchMove={[Function]}
            onTouchStart={[Function]}
            role="button"
            tabIndex="0"
          >
            <span
              className="MuiButtonBase-root-51 MuiIconButton-root-45 MuiPrivateSwitchBase-root-124 MuiCheckbox-root-118 MuiCheckbox-colorPrimary-122"
              onBlur={[Function]}
              onFocus={[Function]}
              onKeyDown={[Function]}
              onKeyUp={[Function]}
              onMouseDown={[Function]}
              onMouseLeave={[Function]}
              onMouseUp={[Function]}
              onTouchEnd={[Function]}
              onTouchMove={[Function]}
              onTouchStart={[Function]}
              tabIndex={null}
            >
              <span
                className="MuiIconButton-label-50"
              >
                <svg
                  aria-hidden="true"
                  className="MuiSvgIcon-root-54"
                  focusable="false"
                  role="presentation"
                  viewBox="0 0 24 24"
                >
                  <path
                    d="M19 5v14H5V5h14m0-2H5c-1.1 0-2 .9-2 2v14c0 1.1.9 2 2 2h14c1.1 0 2-.9 2-2V5c0-1.1-.9-2-2-2z"
                  />
                </svg>
                <input
                  checked={false}
                  className="MuiPrivateSwitchBase-input-127"
                  data-indeterminate={false}
                  onChange={[Function]}
                  tabIndex={-1}
                  type="checkbox"
                />
              </span>
            </span>
            <div
              className="MuiListItemText-root-128"
            >
              <span
                className="MuiTypography-root-63 MuiTypography-subheading-70 MuiListItemText-primary-131"
              >
                Liv Aasen
              </span>
            </div>
            <span
              className="MuiTouchRipple-root-134"
            />
          </div>
        </ul>
      </div>
    </div>
  </div>,
]
`;
